### PR TITLE
IDE-3580 create maven module project by using blade cli

### PR DESCRIPTION
**Pull request by jtydhr88**

### CODE REVIEW

**Comment by JoyeLuo** [comment]
Hey Greg,
Maven and Gradle module fragment junit tests are writen in MavenModuleFragmentProjectTests which are using NewModuleFragmentOp.
And we have the junit tests for all project types now.

**Comment by yanan** [comment]
Build Successful.
 218 tests run, 14 skipped, 0 failed.
Test Passed on Jenkins http://ide-build-1:9090/ on this commit e78d0ef74f0206e11ce123c4bf0928bdc9db2ad0 on build 62

Refer to https://testing-liferayide.wedeploy.io/ui-testing-reports/62/checker.log for detailed testing results

**Comment by yanan** [comment]
Test Passed on Jenkins http://ide-build-1:9090/ on this commit e78d0ef74f0206e11ce123c4bf0928bdc9db2ad0 on build 62

Refer to https://testing-liferayide.wedeploy.io/ui-testing-reports/62/checker.log for detailed testing results

File: maven/plugins/com.liferay.ide.maven.core/src/com/liferay/ide/maven/core/pref/MavenArchetypePreferenceInitializer.java [hunk 1]
```diff
@@ -70,84 +70,6 @@ public void initializeDefaultPreferences() {
 
 		// new project template archetypes
 
-		defaultPrefs.put(
-			LiferayMavenCore.PREF_ARCHETYPE_PROJECT_TEMPLATE_ACTIVATOR,
```
Comment (gamerson) [inline maven/plugins/com.liferay.ide.maven.core/src/com/liferay/ide/maven/core/pref/MavenArchetypePreferenceInitializer.java]: Can we remove these preference keys from the `LiferayMavenCore` class now as well?

Comment (JoyeLuo) [reply maven/plugins/com.liferay.ide.maven.core/src/com/liferay/ide/maven/core/pref/MavenArchetypePreferenceInitializer.java]: yes, I will remove them and resend

**Pull request closed without merging**